source/libs/page-detect.ts

```diff
@@ -372,6 +374,22 @@ export const _isSingleTagPage = [
 	'https://github.com/sindresorhus/refined-github/releases/tag/0.2.1'
 ];
 
+export const isPulse = (): boolean => /^pulse/.test(getRepoPath()!) &&
+	location.pathname.split('/')[3] === 'pulse' &&
+	document.title.startsWith('Pulse ·');
```

Comment:
I don't think this is necessary, `getRepoPath` already returns the 3rd item. There are no other `a/b/pulse` pages

Reply:
Both lines are selected; neither one is ne edsa are you

Reply:
I added the second one to avoid selecting `'https://github.com/sindresorhus/pulse'`

I think this PR is beyond my JS understandings. I think I should close it and wait until someone implements these page detects.

There are a few more page detects for `isActions` that need to be added. One is for when you are searching and one is when clicking on a run.

Reply:
Don’t just use it blindly. Run getRepoPath() on a random page and see what it returns. `fregante/pulse` is not included in its `return` at all.

Reply:
>I think this PR is beyond my JS understandings. I think I should close it and wait until someone implements these page detects.

I don's understand what i am doing here (I promise). I was just copy any pasting.